gittest/gittest.go

```diff
@@ -118,3 +118,18 @@ func WriteFile(t *testing.T, fs billy.Filesystem, path, content string) {
 	err = file.Close()
 	require.NoError(t, err)
 }
+
+func BasicAuthMW(username, password string) func(http.Handler) http.Handler {
+	return func(next http.Handler) http.Handler {
+		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			if username != "" && password != "" {
```

Comment:
Wouldn't this mean that if you pass either `username` or `password` then auth will be bypassed?

```suggestion
			if username != "" || password != "" {
```

Reply:
whoops!